clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterRegistration.java

```diff
@@ -149,20 +152,29 @@ public void execute() {
     }
   }
 
-  private void dealIsSwaggerDifferent(Microservice newMicroservice) {
-    if (isListEquals(newMicroservice.getSchemas(), microservice.getSchemas())) {
-      return;
+  private boolean isSwaggerDifferent(Microservice newMicroservice) {
+    if (!isListEquals(newMicroservice.getSchemas(),microservice.getSchemas())){
+      return coverAndIgnore();
+    }
+    List<SchemaInfo> schemaInfos =serviceCenterClient.getServiceSchemasList(newMicroservice.getServiceId(),true);
+    if (!isListEquals(schemaInfos,this.schemaInfos)){
+      return coverAndIgnore();
     }
```

Comment:
Do not compare swaggers content. We can not guarantee same content for same code.